tests/Unit/Enumerables/JobStatusTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tests\Unit;
+
+use EthicalJobs\SDK\Enumerables\JobStatus;
+use Tests\TestCase;
+
+class JobStatusTest extends TestCase
+{
+    /**
+     * @test
+     * @group Unit
+     */
+    public function testFunctionOutputMatchesConst()
+    {
+      $this->assertEquals(JobStatus::DRAFT, JobStatus::DRAFT());
+      $this->assertEquals(JobStatus::PENDING, JobStatus::PENDING());
+      $this->assertEquals(JobStatus::APPROVED, JobStatus::APPROVED());
+    }
+
+    /**
+     * @test
+     * @group Unit
+     */
+    public function testCanItterateOverConst()
+    {
+      foreach(JobStatus::all() as $value) {
```

Comment:
@bambii7 this is a weak test, because if `all()` yields no results - the test will 'pass' but there was no assertions.
Can you add an assertion in to ensure `all()` yields the results you expect?
Typically you would do this with an
```
$allJobStatuses = JobStatus::all();
$expected = ['DRAFT' => 'DRAFT']; // and so forth

$this->assertSame($expected, $allJobStatuses);
```